src/java.net.http/share/classes/jdk/internal/net/http/Stream.java

```diff
@@ -35,6 +35,7 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Locale;
```

Comment:
Is this new import for Locale really necessary? It seems to me that Locale is not used in the class?

Reply:
Good catch! Removed.